content/departments/ce-support/support/process/support-prioritization.md

```diff
@@ -2,33 +2,47 @@
 
 We have many factors to consider in order to determine priorities and as such, we need very clear boundaries and definitions to ensure there is only ever a single number one priority and it’s straightforward to determine what that is.
 
+## Support exceptions
```

Comment:
To be more clear about this change I'd remove this: We **no longer** offer tier 1 support for pre-sales/trial customers.


And instead say this:
Support is no longer the first-line on pre-sales technical issues. Support may be engaged by CE on specific pre-sales technical issues, as needed by the CE, but Support will not monitor trial slack channels or lead reactive technical troubleshooting.

I'd also add this subsection:

### Engaging Support on pre-sales technical needs

If a CE needs to open a ticket on behalf of a customer, they should do so by emailing support@sourcegraph.com and sharing the following information:
- what is the issue
- what was the prospect attempting to do and what is the expected result
- what steps have been taken to resolve the issue already
- impact of the issue (is it blocking full use, select users, etc)
- link to the Technical Design Document
- context about where in the sales cycle the prospect is and any key dates to be aware of in resolving the issue

Note: if Support picks up a ticket they may temporarily add themselves to the trial slack channel to assist with the issue. However, they would then exit the channel once the issue is resolved.

### Determining severity of pre-sales issues that are elevated to Support

Based on the needs of the ticket, Support will assign a severity level to the ticket based on the following criteria:
- p0: failed deployment or total loss of service
- p1: core features are unavailable or the service as a whole is slow
- p2: non-blocking technical asks or problems

Reply:
This is awesome! Happy to get these changes updated to the page.